src/strategies/ocean-dao-brightid/index.ts

```diff
@@ -0,0 +1,57 @@
+import { getScoresDirect, multicall, getProvider } from '../../utils';
+
+export const author = 'trizin';
+export const version = '0.1.0';
+
+const abi = [
+  'function isVerifiedUser(address _user) external view returns (bool)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const response = await multicall(
+    options.brightIdNetwork,
+    getProvider(options.brightIdNetwork),
+    abi,
+    addresses.map((address: any) => [
+      options.registry,
+      'isVerifiedUser',
+      [address]
+    ]),
+    { blockTag: 'latest' }
+  );
+  let scores = await getScoresDirect(
+    space,
+    options.strategies,
```

Comment:
we should limit these strategies passed into this strategy. else this could increase the load on the server (may be one or two strategies should be good, @bonustrack any suggestions on the number? )

Reply:
Yes @trizin we are looking to have a limit on those strategies too. but should find a way to make sure existing proposals work well :)

Reply:
That's great. I wanted to point it out just in case you missed them. We planned to use this strategy as a master strategy with the five strategies in the `examples.json.` The strategies are lightweight and don't take much time to execute. Perhaps it is better to focus on the execution time rather than the number of the strategies.

Reply:
@trizin In that case it would be better to hard code these strategies inside the strategy than through the options because in the future some other teams may want to use this and may pass different complex strategies. Thanks for understanding :)